conanfile.py

```diff
@@ -43,7 +43,8 @@ class QtConan(ConanFile):
     _submodules = _getsubmodules()
 
     generators = "pkg_config"
-    name = "qt"
+    name = "Qt"
```

Comment:
I had to convert the recipe back to `Qt` because our forked version in the conan-recipes directory was forked from conan-qt originally.  However, the name was changed to `Qt`.  Mac is not a case-sensitive file system so conan doesn't like it when two different recipes use the same name 'qt' vs 'Qt'